include/lsst/pex/logging/BlockTimingLog.h

```diff
@@ -35,7 +35,6 @@
 
 #include <sys/time.h>
 #include <sys/resource.h>
-#include <boost/scoped_ptr.hpp>
```

Comment:
That depends on our policy on transitive includes. I generally think it is better to include all that is used so I added the include for memory and string (which is also used but not included) now.